find-missing.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const bcd = require('mdn-browser-compat-data');
+const tests = require('./generated/tests.json');
+
+const traverseFeatures = (obj, identifier) => {
+  const features = [];
+
+  for (const i in obj) {
+    if (
+      !!obj[i] &&
+      typeof obj[i] == 'object' &&
+      i !== '__compat'
+    ) {
+      if (obj[i].__compat) {
+        features.push(`${identifier}${i}`);
+      }
+
+      features.push(...traverseFeatures(obj[i], identifier + i + '.'));
+    }
+  }
+
+  return features;
+};
+
+const findMissing = () => {
```

Comment:
Good stuff! From that list one can see a lot that can be added to non-standard.idl. But it's best to do so separately from this overhaul.

Reply:
Done in https://github.com/foolip/mdn-bcd-collector/pull/422.